README.md

```diff
@@ -1 +1,6 @@
-# TeslaRIS-backend
\ No newline at end of file
+# TeslaRIS-backend
+
+## Installation
+Installation is very easy, first, position yourself into the project's root directory. After that, simply run: ```mvn spring-boot:run``` which will run your application.
+
+Alternatively, if you are using Intellij IDE, it is as easy as simply running ```TeslarisStarter.java``` file. 
```

Comment:
This is not installation, this is running of the application

Reply:
You are right, i need to make a small edit to pom.xml to make the app executable from project working directory. (https://docs.spring.io/spring-boot/docs/2.0.6.RELEASE/reference/html/deployment-install.html)